load/Validate.py

```diff
@@ -112,23 +112,23 @@ def validateLPTS(self, filesets):
 			logger = Log.getLogger(inp.filesetId)
 			if inp.typeCode == "text":
 
-				scriptName = inp.lptsRecord.Orthography(inp.index)
+				scriptName = inp.languageRecord.Orthography(inp.index)
 				if scriptName != None and scriptName not in self.scriptNameSet:
 					logger.invalidValues(inp.stockNum(), "_x003n_Orthography", scriptName)
 
-				numeralsName = inp.lptsRecord.Numerals()
+				numeralsName = inp.languageRecord.Numerals()
 				if numeralsName != None and numeralsName not in self.numeralsSet:
 					logger.invalidValues(inp.stockNum(), "Numerals", numeralsName)
 
 
 if (__name__ == '__main__'):
 	from DBPLoadController import *
-	from LPTSExtractReader import *
+	from LanguageReader import *
```

Comment:
We should import the LanguageReaderCreator in the main to execute the Validate.py file locally. e.g.

```python
from LanguageReaderCreator import LanguageReaderCreator
```